controller/volume_controller.go

```diff
@@ -2322,9 +2333,10 @@ func (vc *VolumeController) checkAndInitVolumeRestore(v *longhorn.Volume) error
 	if err != nil {
 		return err
 	}
-	backup, err := backupTarget.GetBackup(v.Spec.FromBackup)
+
+	backup, err := backupTarget.InspectBackupConfig(v.Spec.FromBackup)
```

Comment:
Can we directly use the Backup or BackupVolume CR instead? Then we can prevent other components except for backup related controllers to call the backup APIs.

Reply:
Sure, lots of files using backup APIs, I'll update it.